web/jest.config.ts

```diff
@@ -8,6 +8,8 @@ export default async (): Promise<Config.InitialOptions> => {
     setupFilesAfterEnv: ["./setupTests.js"],
     testEnvironment: "jsdom",
     testPathIgnorePatterns: ["<rootDir>/.next/", "<rootDir>/node_modules/", "<rootDir>/cypress/"],
+    rootDir: "./",
+    moduleDirectories: ["node_modules", "<rootDir>"],
```

Comment:
What is this changing for us?

Reply:
This is necessary, because if we're using the seo.config.js to grab the pre fix of `Navigator.nj.gov` the file is in the root directory and our jest tests can't actually see that file unless the root directory is a moduleDirectory

the rootDir declaration is just a good habit and the node_modules are necessary because I believe it implicitly grabs them unless you declare this moduleDirectories field in which case you have to be explicit.